[2023-09-04T06:10:14.931+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2023-04-28T00:00:00+00:00 [queued]>
[2023-09-04T06:10:14.951+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2023-04-28T00:00:00+00:00 [queued]>
[2023-09-04T06:10:14.952+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:10:14.993+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2023-04-28 00:00:00+00:00
[2023-09-04T06:10:15.003+0000] {standard_task_runner.py:57} INFO - Started process 9345 to run task
[2023-09-04T06:10:15.009+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'delete_data_from_table', 'scheduled__2023-04-28T00:00:00+00:00', '--job-id', '2157', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp0v494hi8']
[2023-09-04T06:10:15.014+0000] {standard_task_runner.py:85} INFO - Job 2157: Subtask delete_data_from_table
[2023-09-04T06:10:15.359+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2023-04-28T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:10:15.594+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2023-04-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-28T00:00:00+00:00'
[2023-09-04T06:10:15.596+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2023-04-28' and dag_id = 'dag_with_postgres_operator_v04';
        
[2023-09-04T06:10:15.616+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:10:15.635+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:10:15.654+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2023-04-28' and dag_id = 'dag_with_postgres_operator_v04';
        , parameters: None
[2023-09-04T06:10:15.662+0000] {sql.py:427} INFO - Rows affected: 0
[2023-09-04T06:10:15.683+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=delete_data_from_table, execution_date=20230428T000000, start_date=20230904T061014, end_date=20230904T061015
[2023-09-04T06:10:15.753+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:10:15.783+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
